index.html

```diff
@@ -1,17 +1,76 @@
-<!DOCTYPE html> 
+<!DOCTYPE html>
 <html>
-    <head>
-        <meta charset="utf-8" />
-        <title>Leiden Software Developers</title>
-    </head>
-    <body>
-        <h1>Leiden Software Developers</h1>
-        <p>Future home of the Leiden Software Developers group.</p>
-        <p>Join us at:</p>
-        <ul>
-            <li><a href="https://www.meetup.com/leidendevs/">Meetup</a></li>
-            <li><a href="https://leidendevs.slack.com/">Slack</a></li>
-            <li><a href="https://old.reddit.com/r/LeidenDevs">Reddit</a></li>
-        </ul>
-    </body>
+
+<head>
+    <meta charset="utf-8" />
+    <title>Leiden Software Developers</title>
+    <style>
+        * {
+            font-family: -apple-system, BlinkMacSystemFont, avenir next, avenir, helvetica neue, helvetica, Ubuntu, roboto, noto, segoe ui, arial, sans-serif;
+            color: #111;
+        }
+
+        html {
+            font-size: 18px;
```

Comment:
For accessibility reasons, the root font size should preferably be relative to the user setting. Since all popular browsers have a default font size of `16px`, you can just scale it from there to `18px` with `112.5%` font size. If my eyes are bad or my screen/tv is far away, I might have increased the default font size. Now we respect the users preferences.

Reply:
Instead of using percentages you might want to use `rem` :)

With a base font size of 16px the correct value would be `1.125rem`

Reply:
Yeah, that would work too. In this case it would be exactly the same though, right? Would you say you prefer `rem` in this case?

Reply:
Good catch :)